plugins/link/lang/en.js

```diff
@@ -13,6 +13,7 @@ CKEDITOR.plugins.setLang( 'link', 'en', {
 		title: 'Anchor Properties',
 		name: 'Anchor Name',
 		errorName: 'Please type the anchor name',
+		errorWhitespace: 'Anchor name cannot contain space character',
```

Comment:
There is some small discrepancy between this message and the message in the manual test. IMO the version from the test ("space character**s**") sounds better than the version in language files ("space character").

Probably it's safer to consult it with our docs team to ensure that the grammatical form of the message is ok 🤔 It would be much harder to do that _after_ the release.

Reply:
After talking with @godai78 we have 3 possible options:

1. `Anchor name cannot contain space characters` -> your version
2. `Anchor name cannot contain any spaces` -> this may be not obvious
3. `Anchor name cannot contain any space characters`

For me, 1 and 3 look equally good to me, WDYT?